CHANGELOG.md

```diff
@@ -1,5 +1,616 @@
 # Changelog
 
+## 12.0.0
+
+Teleport 12 brings the following marquee features and improvements:
+
+- Device Trust (Preview, Enterprise only)
+- Non-AD support for Desktop Access (Preview, Enterprise only)
```

Comment:
```suggestion
- Passwordless Windows access for local users (Preview, Enterprise only)
```

Reply:
I think a regression happened here.

Reply:
Hmm yeah not sure what happened, I might have force-pushed without pulling locally. I'll update these again.